sdk/log/crate/src/logger.rs

```diff
@@ -91,6 +95,9 @@ impl<const BUFFER: usize> Logger<BUFFER> {
     }
 
     /// Check if the buffer is empty.
+    // JC nit: this was a bit confusing since `slice.is_empty()` means nearly
+    // the opposite, ie. BUFFER == 0 rather than offset == 0. Is this even
+    // used? If not, I'd recommend removing it or coming up with a better name
```

Comment:
Now that we renamed `offset` to `len`, would it make sense to keep it? It is not used directly by the logger but might be useful for clients.

Reply:
I'd just remove it if it isn't used -- let someone ask for it